### PR TITLE
docs: update sizes and spacing theme example

### PR DESCRIPTION
**Pull request by austinhung81**
For now, I use same example for sizes and spacing themes. 
I think the visual designer will make a new example design for it. I’ll follow up when they have an update.

### CODE REVIEW

File: packages/styled-docs/pages/spacing.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ your project. By default these spacing value can be referenced by the `padding`,
 
 <ThemeParser theme="space" />
 
-By default, the values are proportional, so 1 spacing unit is equal to
+By default, the values are proportional, so 1x spacing unit is equal to
```
Comment (cheton) [inline packages/styled-docs/pages/spacing.mdx]: → `1x` spacing unit

**Comment by cheton** [comment]
這兩個範例看要不要把 `.5rem` 改成 `4x`，看起來比較不會擁擠

#### Width
```jsx
<Stack direction="column" spacing="4x">
```

#### Height
```jsx
<Stack direction="row" spacing="4x">
```

**Review by cheton — APPROVED** [review]

File: packages/styled-docs/pages/spacing.mdx (hunk 1 above)
Comment (austinhung81) [reply packages/styled-docs/pages/spacing.mdx]: 歐！你是說要加``這符號～

Comment (austinhung81) [reply packages/styled-docs/pages/spacing.mdx]: fixed in 9e9be44

**Comment by austinhung81** [comment]
> 這兩個範例看要不要把 `.5rem` 改成 `4x`，看起來比較不會擁擠
> 
> #### Width
> ```js-jsx
> <Stack direction="column" spacing="4x">
> ```
> 
> #### Height
> ```js-jsx
> <Stack direction="row" spacing="4x">
> ```

fixed in 9e9be44

**Pull request merged**